pkg/feature/feature.go

```diff
@@ -40,6 +40,13 @@ func (fs *Flags) Enable(f Flag) {
 	fs.m.Unlock()
 }
 
+// Disable a feature flag.
+func (fs *Flags) Disable(f Flag) {
```

Comment:
We won't need a disable method at this level. If we don't enable it will be disabled and we can do the defaulting at the flag level. See Composition functions handling beta: https://github.com/crossplane/crossplane/blob/75e39029366de2946c9c54f318f7822a6cefcc40/cmd/crossplane/core/core.go#L113

Reply:
I also checked this at the beginning, but in the end I decided to do it this way because controller-runtime is a library. So if we want to have the flag set to true, we need to  [activate it by default](https://github.com/upbound/provider-aws/blob/6bbeee5ef06a4c83eda9f601678b2f425ba082c7/hack/main.go.tmpl#L50) in every provider that currently exists, and every future provider. While for composition functions we just needed to do it in one place.

Thats why I thought its better to have it in the library by default, and then let providers opt-out. But now when I look at it again, it can be confusing that a feature flag is set in the reconciler. Ill switch to what you suggest and just change the Alpha -> Beta thing here

Reply:
I see. 

Considering that the flag already exists in the existing providers controlling the behavior (and we will anyway need a change on every provider), I suggest continuing to use it and also being consistent with Core Crossplane. Updating our template repositories accordingly should be good enough for the new ones.

Reply:
Once we graduate to v1, where disabling the feature wouldn't be possible, we can do a clean-up here and always on by ignoring whatever is passed from the providers.